Bayesian Regression/README.md

```diff
@@ -0,0 +1,94 @@
+# Markov chain Monte carlo
+## Combining these two methods,Markov chain and  monte carlo,above random sampling of high-dimensionality probability distributions that honors the probability dependence between samples by contributing a Markov Chain that comprise the Monte carlo sample.Specifically,MCMC is for performing inference for probability distributions where independent samples from the distributions cannot be drawn or not drawn easily.The idea is that the chain will settle on(find equilibrium) to the desired quantity we are infering.
+## Yet,we are still sampling from the target probability distributions with the goal of approximating a desired quantity, so it ia appropriate to refer to the resulting collection of samples as a monte carlo sample,e.g.  extend of samples drawn often from one long Markov chain
+## There are many markov chain monte carlo algorithms that mostly define different ways of constructing the markov chain when performing each Monte carlo sample.Some of them include Gibbs sampling Algorithms,Metropolis-Hashing-algorithm and so on and o forth.
+# Bayesian Linear Modeling Application
+The basic procedure for implementing Bayesian Linear Modellling includes:
+1.Specifiying priors for the model parameters( normal distributions preferable <math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
+  <mi>N</mi>
+  <mrow data-mjx-texclass="INNER">
+    <mo data-mjx-texclass="OPEN">(</mo>
+    <mi>&#x3BC;</mi>
+    <mo>,</mo>
+    <msup>
+      <mi>&#x3C3;</mi>
+      <mn>2</mn>
+    </msup>
+    <mo data-mjx-texclass="CLOSE">)</mo>
+  </mrow>
+</math>) 
+
+2.Create a model mapping the training inputs to the training outputs,and 
+3. have a Markov Chain maonte carlo (MCMC) algorithm to draw samples from the posterior distributions for the model parameters.
+The end result will be posterior distribution for the parameters.
+## When we want show the linear fit from a Bayesian model, instead of showing only estimate, we can draw a range of lines, with each one representing a different estimate of the model parameters. As the number of datapoints increases, the lines begin to overlap because there is less uncertainty in the model parameters.
+![1_8bA09THSC_Cy5LeijM8oEA](https://user-images.githubusercontent.com/70088281/111058347-02bf4000-84b4-11eb-94d4-1f008040f470.png) 
+![1_C8eR-V648On7Nb11eMqWlg](https://user-images.githubusercontent.com/70088281/111058375-369a6580-84b4-11eb-8913-202339fe03d8.png)
+##When using less data points, the fits have a lot of variance, which means that the model is more unpredictable. Since the priors are washed out by the likelihoods from the data, the OLS and Bayesian Fits are virtually similar with all of the data points.
+##When predicting the output for a single datapoint using our Bayesian Linear Model, we also do not get a single value but a distribution.
+![1_vKbWqDqfz_crZ1C2Ew9dxA](https://user-images.githubusercontent.com/70088281/111058448-b9bbbb80-84b4-11eb-9d9c-c39495874b6d.png)

```

Comment:
```suggestion
![1_8bA09THSC_Cy5LeijM8oEA](https://user-images.githubusercontent.com/70088281/111058347-02bf4000-84b4-11eb-94d4-1f008040f470.png) 

![1_C8eR-V648On7Nb11eMqWlg](https://user-images.githubusercontent.com/70088281/111058375-369a6580-84b4-11eb-8913-202339fe03d8.png)

When using less data points, the fits have a lot of variance, which means that the model is more unpredictable. Since the priors are washed out by the likelihoods from the data, the OLS and Bayesian Fits are virtually similar with all of the data points.<br />

When predicting the output for a single datapoint using our Bayesian Linear Model, we also do not get a single value but a distribution.

![1_vKbWqDqfz_crZ1C2Ew9dxA](https://user-images.githubusercontent.com/70088281/111058448-b9bbbb80-84b4-11eb-9d9c-c39495874b6d.png)

```